.config/CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+## Branch
+Default branch is `dev`. You should set `dev` as the target branch of PRs. `dev` will be merged into `main` regularly after testing.
```

Comment:
Yeah, we should wait David to confirm. I want to clarify that the branching policy (`main` for release, `dev` for developing and testing) is required for the Dev Kit development. If we want to discuss other branching policy, we should also consider the feasibility and effort of code implementation of Dev Kit.

Reply:
Pinging @davidchesnut since he's back from vacation.

Reply:
@hermanwenhe, the `dev` branch works for us. However, can we split this into one PR per sample? It's hard to review the entire thing (GitHub lags).

Reply:
@hermanwenhe is the dev branch for preview purposes in the devkit? If so that totally makes sense to have a dev branch. Or is there some other dependency for having the dev branch? Thanks!

Reply:
@davidchesnut dev branch is mainly for release purpose. Like samples in Teams Toolkit, they add the samples to the dev branch and do all the testing, then merge dev branch to main branch to release it to TTK developers once a while (usually a month).

@AlexJerabek OK, I can do the split.